src/computer_player.c

```diff
@@ -567,12 +590,71 @@ Deck_t* play_card(const Deck_t* cardAddress, Deck_t** head)
         if (temp == NULL)
             return NULL;
         prev->next = temp->next;
-        printf("Successfully deleted  card (%s, %s). \n", CARD_COLOR_STRING[temp->card.color], CARD_NAME_STRING[temp->card.name]);
+        //printf("Successfully deleted  card (%s, %s). \n", CARD_COLOR_STRING[temp->card.color], CARD_NAME_STRING[temp->card.name]);
         return temp;
 
     }
 }
 
+/**
+ * @brief No playable card, computer player reuqests a new card
```

Comment:
Can you change this to "Computer requests a new card when it has no playable card on hand"? Lets be more descriptive.